### PR TITLE
Get rid of networking process and BaseAsyncStopPlugin

### PR DESCRIPTION
**Pull request by cburgdorf**
### What was wrong?

PR #749 unlocks quite some nice refactorings (so only the last four commits of this PR are relevant)

### How was it fixed?

- moved `Node` bootstrapping into syncer plugin and hence got rid of the hardcoded `networking` process. Notice that this does not yet tear sync and peer pool apart (soon ™) 
- removed `BaseAsyncStopPlugin` and related APIs

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [ ] Clean up commit history

[//]: # (For important changes, please add a new entry to the running release notes PR)
[//]: # (You can find the current one using: https://github.com/ethereum/trinity/pulls?q=is%3Aopen+is%3Apr+label%3A%22Release+Notes%22 )
[//]: # (See: https://trinity-client.readthedocs.io/en/latest/contributing.html#pull-requests)
- [ ] Add entry to the [release notes PR](https://github.com/ethereum/trinity/pulls?q=is%3Aopen+is%3Apr+label%3A%22Release+Notes%22)

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

**Comment by cburgdorf** [comment]
@pipermerriam @carver I think you will love this (speaking of the two last commits only)

```
 Get rid of BaseAsyncStopPlugin
 8 files changed, 3 insertions(+), 189 deletions(-)

 Get rid of hardcoded networking process
 5 files changed, 46 insertions(+), 162 deletions(-)
```

**Comment by cburgdorf** [comment]
This has become even nicer :heart_eyes: 

```
Get rid of PlugigManager scopes
4 files changed, 8 insertions(+), 84 deletions(-)
Get rid of BaseAsyncStopPlugin
9 files changed, 3 insertions(+), 195 deletions(-)
Get rid of hardcoded networking process
6 files changed, 52 insertions(+), 168 deletions(-)
```

**Comment by cburgdorf** [comment]
This should be best reviewed after #749

**Comment by cburgdorf** [comment]
@pipermerriam  Just a friendly ping about this. It removes a ton of code (networking process, half of the plugin mess etc) and I'm eager to get it in before I start piling up more stuff on top of it. This builds on top of #749 that should probably be reviewed first.

File: trinity/chains/light_eventbus.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            await self.event_bus.request(event, TO_NETWORKING_BROADCAST_CONFIG)
+        ).account
+
+    async def coro_get_contract_code(self, block_hash: Hash32, address: Address) -> bytes:
+        event = GetContractCodeRequest(block_hash, address)
+        return self._pass_or_raise(
+            await self.event_bus.request(event, TO_NETWORKING_BROADCAST_CONFIG)
+        ).bytez
+
+    def _pass_or_raise(self, response: TResponse) -> TResponse:
```
Comment (pipermerriam) [inline trinity/chains/light_eventbus.py]: This can get moved off the class or made into a `staticmethod`.

**Pull request merged**